### PR TITLE
[custom answers] This patch includes:

### PR DESCRIPTION
**Pull request by nandinimaheshwari7107**
1. Phrases Class that randomly chooses answer from existing library of
    custom phrases
    2. Unit Test for Phrases
    3. Handling of InvalidExpressionException
    4. Flag to silence the custom phrase
    5. Flag to use the default answer phrase

# Pull Request Process

## Before submitting a pull request

> Ask contributors to ensure that any install or build dependencies are removed before the end of the layer when doing a build.

## General steps for completing this pull request

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

### Checklist

> Does your project contributions have steps to follow before being approved? Tests, documentation changing, etc?

Ensure that your `pull request` has followed all the steps below:

- [x] Code compilation
- [ ] All tests passing
- [x] Extended the documentation, if applicable
- [ ] Added myself to the AUTHORS file

### Description

> Ask the contributor to describe his `pull request`.

## Proposed changes

> Ask contributors about why you (or your project maintainers) would accept the pull request. If it solves a previous request, also ask to link to that issue.

### Types of changes

> Show in a checkbox-style, the expected types of changes your project is supposed to have:

- [x] New feature
- [ ] Bugfix
- [ ] So on...

### CODE REVIEW

File: verbcalc/core/answers.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import random
+
+class CustomAnswers:
+    def __init__(self) -> None:
+        self._answers = ['It is', 'The answer is', 'The result is', 'The total is', 'Altogether you get']
+        self._default = 'The result is'
+
+    def get_phrase(self, default:bool = False):
+        return self._answers[random.randrange(0, len(self._answers))] if not default else self._default
```
Comment (ErykPiasecki07) [inline verbcalc/core/answers.py:9]: Didn't know you can do syntax like this, so cool, so Pythonic, absolutely love that line!

File: verbcalc/core/calculate.py [hunk 2]
```diff
@@ -19,16 +20,22 @@ def calculate(sentence: str,
...
     try:
         result = dispatcher.dispatch(translate(sentence).split())
-        if result % 1 == 0:
-            return 'The result is ' + str(int(result))
-        else:
-            return 'The result is ' + str(result)
+        
+        answer = str(int(result)) if result % 1 == 0 else str(result)
+        
+        return answer if silent else ' '.join([CustomAnswers().get_phrase(default=True), answer])
```
Comment (ErykPiasecki07) [inline verbcalc/core/calculate.py:36]: I know it made testing easier, but let's not have `default=True` as library default.

```suggestion
        return answer if silent else ' '.join([CustomAnswers().get_phrase(), answer])
```

**Review by ErykPiasecki07 — CHANGES_REQUESTED** [review]
Great job, I really appreciate the work, amazingly Pythonic and clean code, love extensive testing too! Let's just not make calculate use same answer by default, I know it breaks some other tests that rely on it, I will fix them later.

**Review by ErykPiasecki07 — APPROVED** [review]

**Comment by ErykPiasecki07** [comment]
Also, make sure to add yourself to AUTHORS file if you want as I remember you were asking about it ;)

**Comment by ErykPiasecki07** [comment]
OMG, I am stupid, it was first file, you already did add yourself here, sorry for confusion ;)
Merging it right away!

**Review by KnaByso — APPROVED** [review]
Your code is very clean. Great job!

**Pull request merged**